src/components/_global/BalCheckbox/BalCheckbox.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div :class="['bal-checkbox', wrapperClasses]">
     <div class="flex">
-      <div class="flex items-start">
+      <div class="flex items-center">
```

Comment:
This has implications in other areas when the checkbox label has a large amount of text. For example in high price impact joins, see screenshot:
<img width="389" alt="Screenshot 2023-01-16 at 12 07 01" src="https://user-images.githubusercontent.com/2406506/212674772-4ea54243-9496-476a-985c-3a6a5973f762.png">

Please modify BalCheckbox to take a prop like "align" with a default value of center but that allows for the checkbox to align to the top as well.

Reply:
If the default is center you will have to find all the price impact use cases and add the prop to align with top.

Reply:
fixed